feature_reflect_extension.go

```diff
@@ -200,7 +200,12 @@ func describeStruct(cfg *frozenConfig, typ reflect.Type) (*StructDescriptor, err
 	bindings := []*Binding{}
 	for i := 0; i < typ.NumField(); i++ {
 		field := typ.Field(i)
-		if field.Anonymous && (field.Tag.Get("json") == "" || strings.Split(field.Tag.Get("json"), ",")[0] == "") {
```

Comment:
I disassembled this line into multiple vars because they were used ahead, also makes [line 208](https://github.com/json-iterator/go/pull/121/files#diff-72013d9dc2d76292293510839380f371R208) clearer.